scripts/ci/gitlab/pipeline/test.yml

```diff
@@ -145,6 +145,17 @@ cargo-check-subkey:
     - SKIP_WASM_BUILD=1 time cargo check --locked --release
     - rusty-cachier cache upload
 
+cargo-check-contracts-proc-macro:
+  stage:                           test
+  extends:
+    - .docker-env
+    - .test-refs
+  script:
+    - rusty-cachier snapshot create
+    - cd ./frame/contracts/proc-macro
+    - SKIP_WASM_BUILD=1 time cargo check --locked --release
```

Comment:
It would be good to do this for all crates.

Reply:
Definitely. I didn't intend to take all the crates into account because I'm busy with something else, but I'll open a ticket for it on CI/CD. (https://github.com/paritytech/ci_cd/issues/672)

Do you want me to remove this?

Reply:
Yeah I think we can remove that. It is not a problem of that crate in particular but a general consequence of feature unification.

Reply:
Ok, removed